doc/Makefile

```diff
@@ -1,22 +1,26 @@
-SRCFILE = user_manual
-MANUAL_FILES = user_manual.tex
-LATEX_FLAGS=-interaction=nonstopmode
+SRCFILE=TEAL_user_manual
+# SRC_FILES=../src/HERON_user_manual.tex ../src/HERON_user_manual.bib ../src/Howtorun.tex ../src/Introduction.tex
```

Comment:
I think this line can safely be removed, unless you want it for reference later.